web/src/OverviewTable.tsx

```diff
@@ -277,16 +289,21 @@ function TableEndpointColumn({ row }: CellProps<RowValues>) {
         key={ep.url}
       >
         <StyledLinkSvg />
-        <DetailText>{ep.name || displayURL(ep)}</DetailText>
+        <DetailText title={ep.name || displayURL(ep)}>
+          {ep.name || displayURL(ep)}
+        </DetailText>
       </Endpoint>
     )
   })
-  return <div>{endpoints}</div>
+  return endpoints
```

Comment:
i'm surprised this works? i would have expected this to need a `<>{endpoints}</>` so that it returns a valid react fragment (and is a valid standalone component)

Reply:
I'll revert it to ` <>{endpoints}</>`...I'd removed the containing div while trying to figure out why I couldn't get keyboard focus to work on the endpoint link. (Turns out it may just be an issue with Firefox). But yes, in retrospect this is surprising this worked, even with multiple endpoints!